### DISCUSSION TITLE
Unable to resolve module

### DISCUSSION THREAD

**Discussion by nehalist** [vercel/turborepo#8571] [Help]
### Summary

hey there, I'm encountering some weird issue with a turbo repo:

i do have one app (web) and one package (database). the database package uses drizzle and exposes everything I require for my app. my app uses next-auth (with the drizzle-adapter).

when I now try to go to the login I get an error about drizzle adapter not being able to resolve the drizzle-orm module, which makes sense, since it's only available in the database package. but if I install drizzle-orm to the web app, it still doesn't work - and I have no idea why.

what I've seen is that all my auth related packages (like @auth/drizzle-adapter) are located in the root node_modules directory, where the drizzle-orm (installed via npm i drizzle-orm --workspace=web) is located in the webs node_modules directory - which is most likely the cause of my issue. but why is this even happening?

thanks a lot for any help!

**Comment by anthonyshew** [comment]
Just to provide some context, module resolution is taken care of by the package manager, not Turborepo.

Module resolution is a bit complex of a topic and hard to debug with this amount of information. If you could provide a reproduction, we might be able to help better.
